www/docs/cookbooks/semantic-release.md

```diff
@@ -3,10 +3,33 @@
 GoReleaser does not create any tags, it just runs on what is already there.
 
 You can, though, leverage other tools to do the work for you, like for example
-[svu](https://github.com/caarlos0/svu):
+[svu](https://github.com/caarlos0/svu) or [semantic-release](https://github.com/semantic-release/semantic-release).
+
+## Example: svu
 
 ```bash
 git tag "$(svu next)"
 git push --tags
 goreleaser --rm-dist
 ```
+
+## Example: semantic-release
+
+.releaserc.yml
+
+```yaml
+preset: angular
+plugins:
+  - "@semantic-release/commit-analyzer"
+  - "@semantic-release/release-notes-generator"
+  - "@semantic-release/changelog"
+  - "@semantic-release/git"
+  - - "@semantic-release/exec"
```

Comment:
extra dash here?

Reply:
Yes, that's correct. It is the yaml equivalent of this:

`{
  "plugins": [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    ["@semantic-release/exec", {
      "verifyConditionsCmd": "./verify.sh",
      "publishCmd": "./publish.sh ${nextRelease.version} ${options.branch} ${commits.length} ${Date.now()}"
    }]
  ]
}`

Reply:
ah ok :D